tcp_client.py

```diff
@@ -7,8 +7,27 @@ def start_tcp_client():
     and prints out the response from the server.
     """
 
-    pass
+    client_socket = socket.socket(socket.AF_INET, socket.SOCK_STREAM)\
+    
+    server_address = ('localhost', 65432)
+    print('Starting up on {} port {}'.format(*server_address))
+    client_socket.connect(server_address)
 
+    try:
+        message = b'This is the message. It will be repeated.'
+        print('Sending {!r}'.format(message))
```

Comment:
what is this `{!r}` thing in this format call? can you look it up and explain pls?